### PR TITLE
Error page footer opening times adjustment

### PR DESCRIPTION
**Pull request by georgiaewhitney**
## Who is this for?
For #8327 
Users encountering a status page

## What is it doing for them?
Cleans the footer to only show the opening times link when user visits an error page - was agreed it was not the best solution to render serverData on error pages to show venue opening times in full.

Currently shows:
Screenshot 2023-09-25 at 14 05 45

Proposed:
Screenshot 2023-09-21 at 11 09 56

### CODE REVIEW

File: common/views/components/Footer/index.tsx [hunk 1]
```diff
@@ -174,11 +174,20 @@ const Footer: FunctionComponent<Props> = ({ venues }: Props) => {
...
+              <>
+                <h4 className={font('intb', 5)}>Today&rsquo;s opening times</h4>
+                <OpeningTimes venues={venues} />
+                <Space as="p" v={{ size: 'm', properties: ['margin-top'] }}>
+                  <a href="/opening-times">Opening times</a>
+                </Space>
+              </>
+            ) : (
+              // Error pages do not receive serverData so should only display openingtimes link
+              <Space as="p" v={{ size: 'm', properties: ['margin-bottom'] }}>
```
[the remark this answers is not in the document]
Comment (gestchild) [reply common/views/components/Footer/index.tsx:187]: ah, is it because the spacing value changes?

[the remark this answers is not in the document]
Comment (georgiaewhitney) [reply common/views/components/Footer/index.tsx:187]: Great point! Yes - the spacing drops when the condition is applied. I could look into a cleaner way

[the remark this answers is not in the document]
Comment (gestchild) [reply common/views/components/Footer/index.tsx:187]: Nah, this works

**Review by gestchild — APPROVED** [review]

File: common/views/components/Footer/index.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (georgiaewhitney) [reply common/views/components/Footer/index.tsx:187]: Thanks!

**Pull request merged**